### DISCUSSION TITLE
Add thousands separator for total score in tournament client

### DISCUSSION THREAD

**Discussion by stanrunge** [ppy/osu#15019] [Ideas] [answered]
<img width="858" alt="Screenshot 2021-10-09 at 9 15 34 PM" src="https://user-images.githubusercontent.com/65637274/136671348-5ad64e29-8b3f-45b0-b708-1d58cd9ba4d4.png">

In tournaments, with the score often going into the millions, it can be difficult to read the 2 scores quickly without a thousands separator (ex. 3,000,000 instead of 3000000).

**Comment by peppy** [comment, marked as answer]
If we're going to do it here, we should also consider applying the same change to the main score display in the game. Weirdly, the score counter class supports comma separators but they aren't used *anywhere* right now.

**Reply by peppy** [reply]
Definitely doesn't work well in gameplay if we're leaving the padding zeroes:

![20211010 153502 (dotnet)](https://user-images.githubusercontent.com/191335/136685417-f750d8a6-c654-4e87-baa8-ace031f9b4cc.png)

But in tournament client it seems to be fine, so I'll add it there for now.

**Reply by peppy** [reply]
I've implemented this in https://github.com/ppy/osu/pull/15021.
